suite-common/wallet-core/src/transactions/transactionsReducer.ts

```diff
@@ -180,3 +180,13 @@ export const selectTransactionByTxid = memoizeWithArgs(
         return transactions.find(tx => tx.txid === txid) ?? null;
     },
 );
+
+export const selectTransactionBlockTimeById = memoizeWithArgs(
+    (txid: string, state: TransactionsRootState) => {
+        const transaction = selectTransactionByTxid(txid, state);
+        if (transaction?.blockTime) {
```

Comment:
This could be written shorter. And I think that code readability is not worse with this change.
```suggestion
       return transaction?.blockTime ? transaction.blockTime * 1000 : null;
```

Reply:
I think it's more readable as it was 😅

Reply:
Both is fine. More info here:

https://www.notion.so/satoshilabs/Basic-syntax-49331c82e0d14fb7a1229b6460009e64#ec9c0547aa7a4f8597a0b0bb7f0c8df1

Reply:
Shorter is not always better. When there is calculation inside ternary as well it is harder to read as well as use case - selector has default return at the end which I think is better